apps/lxp/modules/my-training-video-module/section/ContentSection.tsx

```diff
@@ -26,7 +26,8 @@ const ContentSection: FC = (): ReactElement => {
         <div className="w-full">
           {videoModule?.map((item: any, index: number) => {
             console.log("data");
-            const url = item.url.split("https://www.youtube.com/watch?v=").slice(1);
+            // const url = item.url.split("https://www.youtube.com/watch?v=").slice(1);
+            const url = item.url.split("https://youtu.be/").slice(1);
```

Comment:
bikin helper youtube video id extractor

Reply:
export const youtubeVideoIdExtractor = (url: string) =>
  match({ url })
    .with({ url: P.when((url) => url !== undefined) }, ({ url }) => {
      const youtubeRegex = new RegExp(/(?:youtube\.com\/(?:[^/]+\/.+\/|(?:v|e(?:mbed)?)\/|.*[?&]v=)|youtu\.be\/)([^"&?/ ]{11})/);
      const youtubeMatch = url.match(youtubeRegex);
      return youtubeMatch[1];
    })
    .otherwise(() => 'UA0idiwsXZk');